lib/helper/Appium.js

```diff
@@ -135,14 +179,106 @@ class Appium extends WebdriverIO {
 
   _after() {
     if (this.options.restart) return this.browser.end();
-    if (this.options.desiredCapabilities.browserName) {
+    if (this.isWeb && !this.platform) {
```

Comment:
I move this _after hook to WebdriverIO in promises chain fixes, maybe we have to move this change to WebdriverIO helper?